### ISSUE TITLE
Find the ctags file in .git/tags?

### ISSUE THREAD

**Issue by mathie** [atom/symbols-view#77]
I'm not sure where I picked it up from, but I have a git repository template with a post-commit hook that automatically generates a tags file when I commit to, or update the repository. You can find the hooks in my dot-files, [here](https://github.com/mathie/dot-files/tree/master/git_template/hooks).

Would it be possible for symbols-view to pick up this tags file? Couple of things to note:

* the tags file is at `${PROJECT}/.git/tags`; and
* the file references in the tags file are relative to `${PROJECT}/.git` (if that makes sense?) So `ApplicationController`'s entry in the tags file is:

        ApplicationController   ../app/controllers/application_controller.rb    /^class ApplicationController < ActionController::Base$/;"      c

I discovered the latter when I lazily tried symlinking the existing tags file into my project root and it still wouldn't find the symbols. :smile:

**Comment by TrevorS** [comment]
:+1: 

@mathie You probably got it from here:
  http://tbaggery.com/2011/08/08/effortless-ctags-with-git.html.

:)

**Comment by lzap** [comment]
Yeah I do the same, but I don'ŧ use ctags but https://github.com/tmm1/ripper-tags and also automatic tagger https://github.com/lzap/gem-ripper-tags of course with hooks :-)
